### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by rahulyadav139**
Auth modal added. Please review.
Demo: https://gifted-leakey-419629.netlify.app/

### CODE REVIEW

File: src/components/authorization/AuthModal.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.modal-login-show {
+  top: 50%;
+  transform: translateY(-50%) translateX(-50%);
+}
+.modal-signup-show {
+  bottom: 50%;
+  transform: translateY(50%) translateX(-50%);
+} */
```
Comment (sushmita12345) [inline src/components/authorization/AuthModal.css:27]: Suggestion: Remove comment if not need to use

File: src/components/categories/Categories.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+.categories-container input:checked + label {
+  background: var(--primary-color-dark);
+  color: var(--white-color);
+}
+
+.categories-container input:checked + label + div {
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  gap: 1rem;
+} */
```
Comment (sushmita12345) [inline src/components/categories/Categories.css:56]: Same for here

**Comment by sushmita12345** [comment]
- Clean code.
- Easily readable & understandable.
- There is no breakpoint, you can proceed to merge.

**Review by sushmita12345 — APPROVED** [review]

**Review by Debarshi95 — APPROVED** [review]
Looks good to merge. Small suggestion: json and images files can be put into public directory. Json files can be fetched just like we fetch data from an api. Maybe  google about this a bit.

**Pull request merged**